lib/main.dart

```diff
@@ -35,12 +42,12 @@ class MyApp extends StatelessWidget {
       GoRoute(
         path: routePathRootScreen,
         builder: (BuildContext context, GoRouterState state) =>
-            const LoginScreen(),
+        const LoginScreen(),
```

Comment:
Shall we revert this indentation change? Looks like it is causing the CI to fail 😄